Adjusts armory layout and armory

Kasuobes opened the pull request:
Removed: full body armor suits, .45 pistols
Added: armor vests, stun revolvers, tasers, extra flares

Made tactical armory more meaningful, moved grenades, lethals and specialized armor inside.

@Raptor1628 

![](http://i.imgur.com/tXKsKn8.png)

Kasuobes committed:
Adjusts armory layout and armory

Removed: full body armor suits, .45 pistols
Added: armor vests, stun revolvers, tasers, extra flares

Made tactical armory more meaningful, moved grenades, lethals and specialized armor inside.

PsiOmegaDelta commented:
That is a fairly dark corner in the secure armory.

Kasuobes commented:
Now 300% brighter.

![](http://i.imgur.com/JoWtBtH.png)

atlantiscze commented:
Is there a vent/scrubber pair in the upper area? Might be worthwhile to add one if not.

PsiOmegaDelta commented:
Wouldn't hurt to ensure there is a pair in the warden's office as well while you're at it.

Kasuobes commented:
Moved vents/scrubbers from under the tables in tactical armory to plain view, added vents/scrubbers in warden's office hiding under file cabinet and racks. Can't have those roller chairs get stuck in the vents.

![](http://i.imgur.com/BIR4chw.png)

Minijar commented:
I'd suggest replacing those table chargers with wall rechargers, they charge quicker.

Kasuobes commented:
Added wall rechargers, a box of death alarms and spare sec headsets. And a mouse for flavor.

![](http://i.imgur.com/AtASOKf.png)

TheLier commented:
I have one problem. Stun revolvers were something R&D could actually make, and it is one of the few things which can be done without mining department. It also recharges itself, if that counts. I would just prefer if security would ask for it. Science -can- make it, and rather easily in fact.

Honestly, I would rather not see them getting in. Otherwise it all looks nice and dandy.

Kasuobes commented:
Stun revolvers don't recharge themselves, you're thinking of the advanced egun.

The problem with stun revolvers being a science-only item is that sec never asks for it even if RnD is doing stuff. By mapping them into the armory it will at least allow some variety hopefully 10% of the time while emphasizing on non-lethal confrontations. It will also solve the whole issue of sec going "hurr durr can't stun antag they be wearing laser-proof armor must go lethal".

Something noteworthy is that stun revolver "shots" are energy projectiles rather than beams, so they don't go through glass doors or windows.

TheLier commented:
So....if sec does not ask for them should they get it?

Again, that is a reason to ask R&D to make something interesting. All I see that they lose one of the things they can actually do, and that list is very short, science can do very few actually useful things. People should just ask for them, not just get them. 

R&D is already about making power cells for robotistics most of the time. 

If sec wants to get it, just make people aware it exists. This could be something which makes interactions regular, instead of the "science sits on their asses and do nothing most of the time, and even when they do something it usually happens within the department". We need more things from R&D, not less.

And stun revolver is not necessary for play. It is good to have, but you don't have to have it to do your job. 

With the same logic we should give miners a Ripley too from the start.

Kasuobes commented:
RnD has long been boiled down to systematically shoving things into the tech eater to rack up the research level as quickly/efficiently as possible, RnD is anything but interesting in its current state regardless of how much of a dependency to others it is.

While I agree RnD desperately need more gameplay engagement, hamstringing other departments just so they can have a sense of purpose is not the way to go.

Using mining as a terrible example as they already start with two big drills that RnD produces.

TheLier commented:
My point is: Stun revolvers are nice to have but not necessary. Sec can already get by without them. If people just made sec aware of this possiblity, both sides would "win". As more interaction between departments is only good for anyone.

This would actually have an impact on the round, and security could go around and ask for it. Anyone who gets the very basics of R&D can make it.

This is not hamstrining other departments. R&D depends on mining for anything but powercells and mechs. And saying mining depends on research is not quite true in my eyes. You can manually mine anyways, and that is actually doing something. Science in the other hand, is not doing anything, at full time.

If you want to change it, you should stop making changes in the other direction. This makes them even more irrelevant.

tl:dr: Stun revolvers are a nice thing to have, but not necessary at all, and I can say that from experience. Science already can only produce "upgraded" tools, and this is just that. A modifed taser.

Kasuobes commented:
I can likewise make the argument that eguns are nice to have, but not necessary at all since people can just carry a taser and a carbine. Or that carbines aren't necessary at all if everyone has eguns.

Sec already has a tendency to use lethal force sooner than required, gating additional non-lethal options behind another department doesn't help this situation especially when RnD is a role that people rarely willingly play in.

If anything, RnD is suffering not because it's not depended on enough, but rather the whole mechanic of the role is fundamentally boring compared to xenobot, xenobio, viro.

TheLier commented:
Taser is a non-lethal option. They have that. It works just fine most times. 

E-gun is for a situation which might or might not need lethals, as it has two modes.

Carabine is for situation you expect to use lethals, and them only.

Revolver is for....times when the antag has taserbeam but not fully stun proof armour. That is a very niche situation. 

And you know, if RnD is useless, nobody will play it, removing one of the two actually useful things it can make (stun revolver and cyro beaker), is only making it worse. And the "elusive RnD rework", won't come. Frankly if this is our stance, we should give mining Ripley and medical cyrobeakers. Becuse then at least people do not have to "depend" on RnD at all.

This is HRP. It literally takes about ~10 minutes to get the stun revolver. It creates some badly needed interaction, while not having it, is not a problem most of the time, as antags who can't be "bothered" by tasers, often have wide flat resistance over the board.

The only times I can think this could be useful is merc and heist. As a tator can be taken down by sec with tasers, Ninja can't really be bothered, wizard has no armour against taser, changeling is not something affected, malf is a different deal. Again, I do not see it required often. And I used to play sec a lot.

If the issue is that sec does not ask for weapons becuse they think heads would not give them tools they can actually use, then those people should not play heads. Literally, that is being incompetent in a major way, and that -does- break the rules.

Amendment: I got an idea from someone and it sounds like a compromise. Give one revolver to sec instead of three. That way they can use it, but if they feel the need they can ask science. So even in the scenario of an awfully low pop, they can have it, but if the entire department wants them, they can ask R&D to make one. While I'm not the most happy with that, it is better then science never making them.

Kasuobes commented:
RnD will be taken care of from another angle, preferably by revamping the whole system.

comma reviewed (changes requested):
I think having one stun revolver instead of 3 would be good.
As said above, it would remind sec that those exist, while being scarce enough to bug RnD to make them.
Depending on distant and possible rewrite of one of major systems is not good.

Raptor1628 commented:
I'm actually with Lier, thought not from a research standpoint. The stun revolvers shoot a much more powerful stun projectile, on par with the shotgun stun rounds. I'd rather keep them as research-only until energy weapons get an overhaul in general (It's on my list after the Torch)

Kasuobes commented:
Reduced number of stun revolvers to 1 only.

PsiOmegaDelta commented:
Merged with less than 24 hours since the last commit and with no prior approval.
